src/shared/install.c

```diff
@@ -3309,6 +3375,14 @@ int unit_file_get_list(
         return 0;
 }
 
+int mac_nocheck(
+                void *a,
+                const char *b) {
+
+        log_error("CGZONES: mac_nocheck call");
+        return 0;
+}
```

Comment:
Instead of passing this function around, it'd be better to just make `NULL` as the callback mean no check.